src/mon/AuthMonitor.cc

```diff
@@ -1972,9 +1972,12 @@ int AuthMonitor::_check_and_encode_caps(const map<string, string>& caps,
   return 0;
 }
 
-/* Pass both, rdata as well as fmtr, to enable printing of the key after
- * update and set create to True to allow authorizing a new entity instead
- * of updating its caps. */
+/* Update or create an entity, depending on the value of parameter
```

Comment:
Since the comment is being revised, let's use the coding standard way of commenting.

Reply:
use `//` for comments.

Reply:
Done.

I was unaware that multi-line comments (`/* */`) are not allowed. There's no mention CodingStyle. Should we update it?